cmd/thriftbreak/main.go

```diff
@@ -38,6 +39,7 @@ func main() {
 func run(args []string) error {
 	flag := flag.NewFlagSet("thriftbreak", flag.ContinueOnError)
 	gitRepo := flag.String("C", "", "location of git repository. Defaults to current directory.")
+	jsonOut := flag.Bool("json", false, "output as JSON")
```

Comment:
minor: If we're adding a JSON flag, we should maybe mention what fields are included in the JSON object.

    "output as a list of newline-delimited JSON objects with the following fields: File and Message"

(reformat and reword for readability to your preference.)